src/PluginForm/BtcPayRedirectForm.php

```diff
@@ -35,12 +39,11 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
       'cancel_url' => $form['#cancel_url'],
     ];
 
-
-      /** @var \Bitpay\Invoice $btcPayInvoice **/
-      if (! $btcPayInvoice = $payment_gateway_plugin->createInvoice($order, $options)) {
-        $this->redirectToPreviousStep();
-      }
-
+    /** @var \Bitpay\Invoice $btcPayInvoice */
```

Comment:
This is conflicting with my latest commit where I fixed the indentation of this block. Would be great if you could fix it or if you are ok with it I will close this PR and just add the class comment + length fix of the comments.

Question: What is the reason to move the variable assignment out of the `if` statement? (the class hinting works for me in PHPStorm also if it stays as is.)